### PR TITLE
doc: suppored features of kernel client

### PR DESCRIPTION
**Pull request by ukernel**
Fixes: http://tracker.ceph.com/issues/27209
Signed-off-by: "Yan, Zheng" <zyan@redhat.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: doc/cephfs/kernel-features.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+------
+Quota was first introduced by the hammer release. Quota disk format got renewed
+by the Mimic release. Linux kernel clients >= 4.17 can support the new format
+quota. At present, no Linux kernel client support the old format quota.
+
+See `Quotas`_ for more information.
+
+Multiple filesystems within a Ceph cluster
+------------------------------------------
+The feature was introduced by the Jewel releasea. Linux kernel clients >= 4.7
```
Comment (joscollin) [inline doc/cephfs/kernel-features.rst]: s/releasea/release

File: doc/cephfs/kernel-features.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
+
+Supported Features of Kernel Driver
+========================================
+
+Inline data
+-----------
+Inline data was introduced by the Firely release. Linux kernel clients >= 3.19
```
Comment (joscollin) [inline doc/cephfs/kernel-features.rst]: s/Firely/Firefly

File: doc/cephfs/kernel-features.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+can read inline data, can convert existing inline data to RADOS objects when
+file data is modified. At present, Linux kernel clients do not store file data
+as inline data.
+
+See `Experimental Features`_ for more information.
```
Comment (joscollin) [inline doc/cephfs/kernel-features.rst]: `_Experimental Features` missing?

File: doc/cephfs/kernel-features.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+Quotas
+------
+Quota was first introduced by the hammer release. Quota disk format got renewed
+by the Mimic release. Linux kernel clients >= 4.17 can support the new format
+quota. At present, no Linux kernel client support the old format quota.
+
+See `Quotas`_ for more information.
```
Comment (joscollin) [inline doc/cephfs/kernel-features.rst]: Missing `_Quota` ?

**Comment by joscollin** [comment]
jenkins render docs

**Comment by tchaikov** [comment]
@ukernel might want to add this doc to `doc/cephfs/index.rst` ?

**Comment by batrick** [comment]
@ukernel doc build fails. Also please address @joscollin 's comments.

**Comment by batrick** [comment]
render docs please

**Comment by batrick** [comment]
jenkins render docs

**Pull request merged**